src/lecturecut.py

```diff
@@ -153,6 +154,16 @@ def transcode(progress, instance):
   cache_path = CACHE_PREFIX + f"/{instance}/"
   segments = instances[instance]["segments"]
   cuts = instances[instance]["cuts"]
+  # check for argument Timestamps only "--tsonly"
+  if ts_only:
+    #write csv with timestamps
+    with open("timestamps.csv", "w") as f:
+      for cut in cuts:
+        f.write(f"{cut[0]},{cut[1]},\n")
+    exit("timestamps.csv written")
+    
+      
+  
```

Comment:
Separation of Concerns.
The transcode function should not even be called when there is no intention to transcode.